### PR TITLE
Add resources in Uzbek

### PR DESCRIPTION
**Pull request by buzurgmexrubon**
- Add Uzbek ReadMe.uz.md, aspnetcore-developer-roadmap.uz.png and aspnetcore-developer-roadmap.uz.xmind files

- Fix naming issue. Orcherstration -> Orchestration (ReadMe.md)

### CODE REVIEW

File: aspnetcore-developer-roadmap.uz.png
Comment (MoienTajik) [inline aspnetcore-developer-roadmap.uz.png:1]: Could you please compress this PNG file using https://tinypng.com?

**Review by Ar-shid — APPROVED** [review]
Looks great, Thanks

File: aspnetcore-developer-roadmap.uz.png
Comment (buzurgmexrubon) [reply aspnetcore-developer-roadmap.uz.png:1]: **aspnetcore-developer-roadmap.uz.png** file compressed. 

Current size is _649_ kb

**Comment by buzruk** [comment]
✅
aspnetcore-developer-roadmap.uz.png file compressed.

**Review by MoienTajik — APPROVED** [review]

**Pull request merged**